Dockerfile

```diff
@@ -10,17 +10,55 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 USER root
 
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update --yes && \
+    # - `apt-get upgrade` is run to patch known vulnerabilities in system packages
+    #   as the Ubuntu base image is rebuilt too seldom sometimes (less than once a month)
+    apt-get upgrade --yes && \
+    apt-get install --yes --no-install-recommends build-essential gfortran && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+USER ${NB_UID}
+RUN set -x && \
+    # Installing jupyter lab extensions in the base environment
+    mamba install --yes jupyterlab-git=0.44.0 \
+    mamba_gator=5.2.1 \
+    jupytext=1.16.1 \
+    jupyter-archive=3.4.0 \
+    jupyter-server-proxy=4.1.0 \
+    dask-labextension=6.2.0 \
+    ipywidgets=8.1.1 \
+    jupyterlab=3.6.7 \
+    jupyter_bokeh=3.0.7 && \
+    mamba clean --all -f -y
+
+USER root
```

Comment:
You could skip having to switch back to the root user if you copied the file to `/tmp/environment.yml` instead.

Reply:
Done